packages/components/src/range-control/stories/index.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import { number, text } from '@storybook/addon-knobs';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import RangeControl from '../';
+
+export default { title: 'RangeControl', component: RangeControl };
+
+const RangeControlWithState = ( { label, help, min, max, beforeIcon, afterIcon, allowReset, ...props } ) => {
+	const [ value, setValue ] = useState( 5 );
+
+	return (
+		<RangeControl
+			{ ...props }
+			help={ help }
+			label={ label }
+			value={ value }
+			onChange={ ( val ) => setValue( val ) }
```

Comment:
Could we just pass `onChange={ setValue }`?

Reply:
Yes, this has been updated, thank you.